Software/spartan/src/data/mdp.h

```diff
@@ -100,14 +110,14 @@ namespace spartan {
 
     class PayloadHealthPacket : public PacketType {
     public:
+        virtual int getSize() const;
         virtual void populate(const MasterDataPacket &dp);
     private:
         float m_payload_voltage;
         float m_payload_current;
     };
 
     class PayloadDataPacket {};
-     */
 } // namespace spartan
 
 #endif // MDP_H_INCLUDED
```

Comment:
Would prefer these (non-MDP) declarations get moved to `datapacket.h` and the implementations get moved to `datapacket.cpp`; cleaner structure that way.

Reply:
Makes sense--done.